### PR TITLE
Command/openssl

### PR DESCRIPTION
**Pull request by piranna**
Mocks for missing functions needed by `openssl` command.

### CODE REVIEW

File: lib/emscripten/src/lib.rs [hunk 1]
```diff
@@ -816,6 +822,12 @@ pub fn generate_emscripten_env(globals: &mut EmscriptenGlobals) -> ImportObject
         "invoke_viid" => func!(crate::emscripten_target::invoke_viid),
         "invoke_viidii" => func!(crate::emscripten_target::invoke_viidii),
         "invoke_viidddddddd" => func!(crate::emscripten_target::invoke_viidddddddd),
+
+        // ucontext.h
+        "_getcontext" => func!(crate::emscripten_target::_getcontext),
```
Comment (syrusakbary) [inline lib/emscripten/src/lib.rs]: Can we move this functions to `context.rs` (or `ucontext.rs`)?

Comment (piranna) [reply lib/emscripten/src/lib.rs]: Create a new `ucontext.rs` file? That's ok for me :-)

Comment (syrusakbary) [reply lib/emscripten/src/lib.rs]: Yup!

**Pull request merged**